daemon/config_unix.go

```diff
@@ -67,7 +68,7 @@ func (config *Config) InstallFlags(cmd *flag.FlagSet, usageFn func(string) strin
 	cmd.BoolVar(&config.Bridge.EnableIPMasq, []string{"-ip-masq"}, true, usageFn("Enable IP masquerading"))
 	cmd.BoolVar(&config.Bridge.EnableIPv6, []string{"-ipv6"}, false, usageFn("Enable IPv6 networking"))
 	cmd.StringVar(&config.Bridge.IP, []string{"#bip", "-bip"}, "", usageFn("Specify network bridge IP"))
-	cmd.StringVar(&config.Bridge.Iface, []string{"b", "-bridge"}, "", usageFn("Attach containers to a network bridge"))
+	cmd.StringVar(&config.Bridge.Iface, []string{"b", "-bridge"}, bridge.DefaultBridgeName, usageFn("Attach containers to a network bridge"))
```

Comment:
This fix seems to be very specific for the bridge driver. I would encourage you to take a look at the new network driver/plugin design that is currently active in the experimental channel. The actual usage of linux bridge (docker0 in this example) is a driver/plugin concern. Also the assumption of veth pair is incorrect. There are plugins out there they can use OVS bridge instead of linux bridge and the assumptions that you are making for the CRIU design will break.
Hence, i encourage you to try and find a good design and use libnetwork primitives to handle the bridge or veths instead of trying to guess it at a different layer.